tests/resources/VCH-Util.robot

```diff
@@ -26,8 +26,8 @@ Set Test Environment Variables
     Run Keyword If  '${status}' == 'FAIL'  Set Environment Variable  PUBLIC_NETWORK  'VM Network'
     ${status}  ${message}=  Run Keyword And Ignore Error  Environment Variable Should Be Set  TEST_DATACENTER
     Run Keyword If  '${status}' == 'FAIL'  Set Environment Variable  TEST_DATACENTER  ${SPACE}
-    ${status}  ${message}=  Run Keyword And Ignore Error  Environment Variable Should Be Set  DRONE_MACHINE
-    Run Keyword If  '${status}' == 'FAIL'  Set Environment Variable  DRONE_MACHINE  'local'
+    ${status}  ${message}=  Run Keyword And Ignore Error  Environment Variable Should Be Set  DRONE_HOSTNAME
+    Run Keyword If  '${status}' == 'FAIL'  Set Environment Variable  DRONE_HOSTNAME  'local'
```

Comment:
I don't think this logic is correct any more now DRONE_HOSTNAME is not used.
I don't think it will cause failure, but doesn't make sense.